components/global/product_switcher.tsx

```diff
@@ -142,6 +143,7 @@ const ProductSwitcher = (): JSX.Element => {
                         aria-label='Select to open product switch menu.'
                     />
                     <ProductSwitcherTip/>
+                    <ProductBranding onClick={handleClick}/>
```

Comment:
Maybe we can make the ProductSwitcherContainer handle the click so we don't have to pass this to two components?

Reply:
i did this to prevent closing of the menu when the divider is being clicked

Reply:
oh, and it is easier to handle the action on the same level as the state for the opening/closing of the menu sits

Reply:
sry ... my previous comments were for a different kind of component ... I mixed those up 🙇🏼‍♂️ 

But the same kinda goes for this as well: We don't want the space to be clickable, but only those that are visually indicating a potential action and by passing it to the container we would have the effect that the whole area is clickable

Reply:
But the only two things in the area are the two things we want clickable. I think it would be a bug if there was a tiny space between them that was unclickable. Presumably we want the whole area to eventually highlight not just the grid button?

Reply:
about the highlighting: that is something for UX to decide upon (although IMO you are right)
the clickable area: it depends ... right now there is only the 2 items with no space in between, so we can certainly apply it to the parent component